### ISSUE TITLE
Bug - Slider: tab order incorrect when input is above thumb

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#5121]
**Describe the problem**
The tab order is incorrect when the value input is placed above the thumb. Rather than tabbing to the input then the thumb slider, focus goes to the thumb slider and then the value input.

Related to https://github.com/patternfly/patternfly-react/issues/8053

**Comment by RuiBrian** [comment]
I can be assigned to this!

**Comment by mcoker** [comment]
@thatblindgeye I'm a little confused after looking at this issue 😅  Looks like the react slider got the update to DOM/tab order on 10/13 from https://github.com/patternfly/patternfly-react/pull/8190. But from the PR @RuiBrian opened, looks like we decided to not make this change in core on 10/20. If I'm following correctly, should this core issue be closed, and the react component back that change out? I looked and didn't see a react issue for that change.
